static/js/mediastream/peercall.js

```diff
@@ -96,6 +105,10 @@ define(['jquery', 'underscore', 'mediastream/utils', 'mediastream/peerconnection
 
 	PeerCall.prototype.onCreateAnswerOffer = function(cb, sessionDescription) {
 
+		// We processed the incoming Offer by creating an answer, so it's safe
+		// to create own Offers to perform renegotiation.
+		this.peerconnection.readyForRenegotiation = true;
```

Comment:
This should probably be a function, rather than directly setting an external member.